platform-operator/controllers/verrazzano/component/opensearch/opensearch_component.go

```
@@ -203,6 +204,26 @@ func (o opensearchComponent) ValidateUpdateV1Beta1(old *installv1beta1.Verrazzan
 	if err := common.CompareStorageOverridesV1Beta1(old, new, ComponentJSONName); err != nil {
 		return err
 	}
+	if old.Spec.Components.OpenSearch == nil {
+		return nil
+	}
+	opensearchOld := old.Spec.Components.OpenSearch
+	if new.Spec.Components.OpenSearch == nil {
+		return nil
+	}
+	opensearchNew := new.Spec.Components.OpenSearch
+
+	numNodesold, _ := GetNodesNumber(opensearchOld)
```

why is it ignoring the error here?

---

and on the next line